src/main/java/chess/domain/strategy/GeneralMoveStrategy.java

```diff
@@ -0,0 +1,33 @@
+package chess.domain.strategy;
+
+import chess.domain.color.Color;
+import chess.domain.piece.Piece;
+import chess.domain.piece.Position;
+import java.util.Map;
+import java.util.Set;
+
+public class GeneralMoveStrategy extends MoveStrategy {
+
+    public GeneralMoveStrategy(Map<Position, Piece> board) {
+        super(board);
+    }
+
+    @Override
+    public void move(Color turnColor, Position from, Position to) {
```

Comment:
```suggestion
    public void move(Color turnColor, Position source, Position destination) {
```
from 과 to 도 좋지만, source / destination 이라고 이름을 지으면 로직을 읽기 더 수월할 것 같네요! 프로젝트 전체적으로 한 번 확인해주세요.

Reply:
좋은 네이밍이네요! 👍👍